static/app/views/projects/index.tsx

```diff
@@ -0,0 +1,13 @@
+import {GettingStartedWithProjectContextProvider} from './gettingStartedWithProjectContext';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export function Projects({children}: Props) {
+  return (
+    <GettingStartedWithProjectContextProvider>
```

Comment:
since the `project/new` endpoint is under `projects` it was needed to create this new page/component, wrapping the children with the "gettingStarted...." context